src/app/contextBroker/CMakeLists.txt

```diff
@@ -88,6 +88,9 @@ ELSEIF(${DISTRO} STREQUAL "openSUSE_13.1")
 ELSEIF(${DISTRO} MATCHES "CentOS_6.*")
   MESSAGE("contextBroker: CentOS DISTRO: '${DISTRO}'")
   TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} -lmicrohttpd -lmongoclient -lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt -lboost_regex-mt)
+ELSEIF(${DISTRO} MATCHES "CentOS_release")
```

Comment:
I should have expected something like this here:

```
ELSEIF(${DISTRO} MATCHES "CentOS_7.*")
```

If `CentOS_release` is the one which works, something makes me think that the procedure to fill DISTRO variable (based on the `osDistro.sh` script) is not working...

Could you tell me the result of the following commands in your CentOS 7.x box, please?

* `cat /etc/redhat-release`
* `scripts/build/osDistro.sh` (from the fiware-orion repository root)